packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -1,8 +1,43 @@
 <template>
     <div
         :class="$style['c-imageTile']"
-        data-test-id="imageTile">
-        I am a ImageTile Component (GB)
+        data-test-id="image-tile-component">
+        <a
+            :class="[
+                $style['c-imageTile-link'], {
+                    [$style['c-imageTile-link--toggle']]: !isLink
+                }]"
+            :href="isLink ? href : '#'"
+            :aria-hidden="!isLink"
+            :tabindex="isLink ? false : -1"
+            data-test-id="image-tile-link">
+            <span class="is-visuallyHidden">
+                {{ displayText }}
+            </span>
+        </a>
+        <input
+            :id="`imageTileToggle-${tileId}`"
+            type="checkbox"
+            class="is-visuallyHidden"
+            :class="$style['c-imageTile-checkbox']"
+            data-test-id="image-tile-input"
+            @change="toggleFilter">
+        <label
+            :class="[
+                $style['c-imageTile-label'], {
+                    [$style['c-imageTile-label--selected']]: isFilterSelected
+                }]"
+            :for="`imageTileToggle-${tileId}`"
+            data-test-id="image-tile-label">
+            <img
```

Comment:
is it worth adding `role="presentation"` as the image doesn't provide any additional information to the display text?

Reply:
I'll add a note to the image ticket 👍